index.html

```diff
@@ -0,0 +1,146 @@
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale =1.0">
+        <title>Portfolio project</title>
+    <link rel="stylesheet" href="styles.css">
+    <script src="https://kit.fontawesome.com/d901e0cb9a.js" crossorigin="anonymous"></script>
+    </head>
+    <body>
+      
+        <header>
+          <nav>
+            <ul>
+              <li><a class="logolink" href=""><b>My Logo</b></a></li>
+              <li style="float:right;" class="dropdown">
+<a href=""><img src="hammenu.png" alt="ham" style="width: 90px;height: 60px;"></a></li>
+             </ul>
+                    </nav>
+          <div class="textdiv"><h1>I’m Alexandra <br>
+            Glad to see you!</h1>
+            <p>I’m a software developer! I can help you build a product , feature or website Look through <br>
+              some of my work and experience! If you like what you see and have a project
+              you need <br> 
+              coded, don’t hestiate to contact me.</p>
+            <section><i class="fa-brands fa-twitter"></i>
+              <i class="fa-brands fa-linkedin-in"></i>
+              <i class="fa-solid fa-m"></i>
+              <i class="fa-brands fa-github"></i>
+              <i class="fa-regular fa-hand-peace"></i></section>
+            </div>
+        </header>
+       <div class="container">
+        <div class="imgdiv">
+            <img src="image1.png" alt="Nature" height="300" width="300">
+        </div>
+        <div class="tonic">
+                <h1>Tonic</h1>
+                <p>CANOPY * <span>Back End Dev</span> * <span>2015</span></p>
+                <br>
+                <p>A daily selection of privately personalized reads; no accounts <br> or sign-ups required.</p>
+                <span class="htcj">HTML</span> 
+                <span class="htcj">CSS</span>
+                <span class="htcj">Javascript</span><br><br>
+                  <div class="btn">
+                    <a href="image1.png" target="_blank">See Project</a>
+                  </div>
+        </div>    
+       </div>
+       <div class="container2">
```

Comment:
- [ ] Since this div's closing tag is the last thing on the headline section, you should have the section's closing tag right after it, before anything else. I am now hoping you clearly understand what the previous reviewer had requested. Please implement the change.